src/propset/calendarPropSet.js

```diff
@@ -88,7 +88,9 @@ export default function calendarPropSet(props) {
 		case '{urn:ietf:params:xml:ns:caldav}schedule-calendar-transp':
 			xmlified.push({
 				name: [NS.IETF_CALDAV, 'schedule-calendar-transp'],
-				value,
+				children: [{
+					value,
+				}],
```

Comment:
```suggestion
				children: [{
					name: [NS.IETF_CALDAV, value],
				}],
```

If I read the RFC correctly. Will test in a second.